wix/Lang/OpenJDK.Base.en-us.wxl.template

```diff
@@ -7,10 +7,10 @@
 	<String Id="OSVersionRequired" Value="Windows 7 or later is required." />
 	<!-- <String Id="NewerVersionInstalled">Newer version of OpenJDK is already installed.</String> -->
 	<String Id="ProductIsNotSupportedOnItanium" Value="This product is not supported on Itanium 64-bit systems." />
-	<String Id="FeatureEnvironmentTitle" Value="Add to PATH" />
-	<String Id="FeatureEnvironmentDescription" Value="Add to PATH environment variable." />
-	<String Id="FeatureJavaHomeTitle" Value="Set JAVA_HOME variable" />
-	<String Id="FeatureJavaHomeDescription" Value="Set JAVA_HOME environment variable." />
+	<String Id="FeatureEnvironmentTitle" Value="Modify PATH variable" />
+	<String Id="FeatureEnvironmentDescription" Value="Modify PATH environment variable by prepending the JDK installation directory to the beginning of PATH." />

```

Comment:
This feels pretty long? Have you built the MSI locally or downloaded one built in the GitHub workflow to see if this actually fits in the box?

Reply:
It should fit based on a visual assessment :P

Reply:
@gdams, here are some screenshots (ignore the logo, I made these locally just to test the visuals of the new wording):

Reply:
![image](https://github.com/user-attachments/assets/680839da-0d37-488d-b785-0b3dc616db1f)

Reply:
![image](https://github.com/user-attachments/assets/3e3ef165-0c2d-4ad0-974b-0b78152ef305)

Reply:
Based on previous test, it must fit:
https://github.com/adoptium/installer/issues/732#issuecomment-1880007153

Reply:
@douph1 are you able to provide the french text updates for this PR?

Also ping @hendrikebbers @CarmenDelgadoEclipse and @zdtsw who may be able to help with the other locales

Reply:
is that to have a PR for https://github.com/brunoborges/installer/blame/patch-1/wix/Lang/OpenJDK.Base.zh-cn.wxl.template on line 10-13?

Reply:
@zdtsw I believe it is easier by looking at the diff of this PR. There are now a total of 5 lines being modified here. They are related to the following `Id` items:

- FeatureEnvironmentTitle
- FeatureEnvironmentDescription
- FeatureJavaHomeTitle
- FeatureJavaHomeDescription
- BrowseDlg_Title

Reply:
ref: https://github.com/adoptium/installer/pull/951